src/apps/viewer/index.ts

```diff
@@ -409,4 +432,32 @@ export interface LoadTrajectoryParams {
     | { kind: 'coordinates-data', data: string | number[] | ArrayBuffer | Uint8Array, format: BuildInStructureFormat },
     coordinatesLabel?: string,
     preset?: keyof PresetTrajectoryHierarchy
-}
\ No newline at end of file
+}
+
+export const ViewerAutoPreset = StructureRepresentationPresetProvider({
```

Comment:
I think this should be part of the extension and not just the viewer.

Reply:
Hmm, I added two presets (pLDDT and qmean) to the extension. I thought of this one as specific to the viewer where we can integrate code from multiple extensions.